src/components/KMultiselect/KMultiselect.vue

```diff
@@ -320,158 +321,49 @@ defineOptions({
-    default: () => ({}),
-  },
-  placeholder: {
-    type: String,
-    default: '',
-  },
-  searchPlaceholder: {
-    type: String,
-    default: '',
-  },
-  kpopAttributes: {
-    type: Object as PropType<Omit<PopoverAttributes, 'target' | 'trigger'>>,
-    default: () => {},
-  },
-  dropdownMaxHeight: {
-    type: String,
-    default: '300',
-  },
-  /**
-   * The width of the multiselect and popover's min-width
-   */
-  width: {
-    type: String,
-    default: '100%',
-  },
-  /**
-   * Number of rows of selections to show when focused
-   */
-  selectedRowCount: {
-    type: Number,
-    default: 1,
-  },
-  /**
-   * Determines whether to show total selected count (false), or
-   * row(s) of selections when collapsed
-   */
-  collapsedContext: {
-    type: Boolean,
-    default: false,
-  },
-  /**
-   * Items are JSON objects with required 'label' and 'value'
-   * {
-   *  label: 'Item 1',
-   *  value: 'item1'
-   * }
-   */
-  items: {
-    type: Array as PropType<MultiselectItem[]>,
-    default: () => [],
-    // Items must have a label & value
-    validator: (items: MultiselectItem[]) => !items.length || (items.every(i => i.label !== undefined && i.value !== undefined) && itemValuesAreUnique(items)),
-  },
-  /**
-   * Override default filter functionality of case-insensitive search on label
-   */
-  filterFunction: {
-    type: Function,
-    default: (params: MultiselectFilterFunctionParams) => params.items.filter((item: MultiselectItem) => item.label?.toLowerCase().includes(params.query?.toLowerCase())),
-  },
-  /**
-   * A flag for autosuggest mode
-   */
-  autosuggest: {
-    type: Boolean,
-    default: false,
-  },
-  /**
-   * Allow creating new items
-   */
-  enableItemCreation: {
-    type: Boolean,
-    default: false,
-  },
-  /**
-   * Loading state in autosuggest
-   */
-  loading: {
-    type: Boolean,
-    default: false,
-  },
-  /**
-   * Dropdown footer text
-   */
-  dropdownFooterText: {
-    type: String,
-    default: '',
-  },
-  /**
-   * Dropdown footer text position
-   * Accepted values: 'sticky' and 'static'
-   */
-  dropdownFooterTextPosition: {
-    type: String as PropType<DropdownFooterTextPosition>,
-    default: 'sticky',
-  },
-  /**
-   * Validator function for item creation.
-   */
-  itemCreationValidator: {
-    type: Function as PropType<(query: string) => boolean>,
-    default: () => true,
-  },
-})
-
-const emit = defineEmits<{
-  (e: 'selected', item: MultiselectItem[]): void
-  (e: 'input', value: string[]): void
-  (e: 'change', item: MultiselectItem | null): void
-  (e: 'update:modelValue', value: string[]): void
-  (e: 'query-change', query: string): void
-  (e: 'item-added', value: MultiselectItem): void
-  (e: 'item-removed', value: MultiselectItem): void
-}>()
+const {
+  modelValue = [],
+  label = '',
+  help = '',
+  error,
+  labelAttributes = {},
+  placeholder = '',
+  searchPlaceholder = '',
+  kpopAttributes = {},
+  dropdownMaxHeight = '300',
```

Comment:
```suggestion
  dropdownMaxHeight = '300px',
```

We currently use both unitless and unit-included CSS length values. Let’s unify them by consistently adding units.